app/code/Magento/Sales/Controller/Adminhtml/Order/AddressSave.php

```diff
@@ -51,4 +126,20 @@ public function execute()
             return $resultRedirect->setPath('sales/*/');
         }
     }
+    
+    /**
+     * Update region data
+     *
+     * @param array $attributeValues
+     * @return array
+     */
+    protected function updateRegionData($attributeValues)
```

Comment:
Why this method protected? Is it referenced from the children classes?

Reply:
@vrann this function is only used in this class. I have updated the visibility to private.